api/graphite.go

```diff
@@ -1040,6 +1048,12 @@ func (s *Server) executePlan(ctx context.Context, orgId uint32, plan *expr.Plan)
 	meta.RenderStats.PlanRunDuration = time.Since(preRun)
 	planRunDuration.Value(meta.RenderStats.PlanRunDuration)
 	span.LogFields(traceLog.Float64("PlanRunMillis", durToMillis(meta.RenderStats.PlanRunDuration)))
+
+	// Log finished stats
+	// TODO - reusable buffer
+	prettyMeta, _ := meta.MarshalJSONFast(nil)
+	log.Infof("executePlan: Completed traceID=%s meta=%s err=%v", traceID, string(prettyMeta), err)
```

Comment:
not logging the request again here? do you plan to associate these stats with the requests with a log analysis tool?

Reply:
Yeah, we use Humio to filter/join on traceID. These logs have been very useful in the short time we've had them. I tried to be cognizant of log volume.